web-ui/src/backup_account/page.js

```diff
@@ -36,13 +37,20 @@ export class Page extends React.Component {
 
   saveBackupEmail = (status) => {
     this.setState({ status });
-  }
+  };
 
   mainContent = () => {
     if (this.state.status === 'success') return <Confirmation />;
     return <BackupEmail onSubmit={this.saveBackupEmail} />;
   };
 
+  showSnackbarOnError = (t) => {
+    if (this.state.status === 'error') {
+      return <SnackbarNotification message={t('backup-account.error.submit-error')} isError />;
+    }
+    return undefined;
```

Comment:
We also thought the same but the linter doesn't like it :) for some weird reason.  Thats why.  The linter error is - `Expected to return a value at the end of arrow function  consistent-return`.